pyecharts/javascripthon/api.py

```diff
@@ -74,6 +75,9 @@ def default(self, obj):
         if isinstance(obj, (datetime.datetime, datetime.date)):
             return obj.isoformat()
 
+        if isinstance(obj, Passport):
```

Comment:
implement `__json__` for `Passport` ,and this encoder will not explicitly import `Passport`.
```
class Passport:
    def __json__(self):
        return self.value

NULL = Passport('json value for Python None', value=None)
```

Reply:
Yes, good point! See updates please.